src/RegistryBuilder.php

```diff
@@ -221,4 +222,14 @@ protected static function initTemperature(Registry $registry): void
         $registry->register('celsius', new Unit(new UnitPart(1, Dimension::TEMPERATURE, 1, 273.15)));
         $registry->register('fahrenheit', new Unit(new UnitPart(5 / 9, Dimension::TEMPERATURE, 1, 459.67)));
     }
+
+    protected static function initFrequency(Registry $registry): void
+    {
+        static::registerSiUnit(
+            $registry,
+            'hertz',
+            ['hz'],
+            new Unit(new UnitPart(86400, Dimension::TIME, -1))
```

Comment:
```suggestion
            new Unit(new UnitPart(1, Dimension::TIME, -1))
```
Otherwise it'd be a yearhertz.